.circleci/config.yml

```diff
@@ -45,17 +45,9 @@ jobs:
       - checkout
 
       - run:
-          command: make linux darwin windows
+          command: make clean linux darwin windows
```

Comment:
clean isn't needed unless a make has already been done... which now in this case it hasn't, right? Does no harm though.